README.md

```diff
@@ -1,2 +1,2 @@
 # api-gateway-verification
```

Comment:
Please could we change the name of the repository? The current name it has doesn't match the library's intent (it's for http request signature verification in general and so could be reused by our components if the need arose) as well as by someone else who might want to sue the same technique.